scripts/generate-ebpf.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: PolyForm
+# Script to generate Go bindings from eBPF C code
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "${SCRIPT_DIR}/.." && pwd)"
+
+echo "Generating eBPF Go bindings..."
+
+# Generate Go bindings for each eBPF program
+cd "${PROJECT_ROOT}"
+
+# Download only the bpf2go tool dependency
+echo "Downloading bpf2go tool..."
+go mod download github.com/cilium/ebpf
```

Comment:
because you already added this package to the go.mod and added github.com/cilium/ebpf/cmd/bpf2go to `tools.go` I believe you don't need to download again.

In fact we could just remove this script and update the make target to simply be:
```make
.PHONY: generate-ebpf-types
generate-ebpf-types: ## Generate Go types from eBPF header files
	@echo "Generating Go types from eBPF headers..."
	go generate $(ROOT)/pkg/ebpf/...
```